component-samples/http-device-sample/src/main/java/org/fidoalliance/fdo/sample/Device.java

```diff
@@ -469,9 +490,34 @@ protected MessagingService getMessagingService(Composite request) {
         WebClient client = new WebClient(url, dr, to2Dispatcher);
         client.run();
       } catch (RuntimeException e) {
-        logger.info("Unable to contact Owner at " + url + ". " + e.getMessage());
+        if (isCausedBy(e, SocketException.class)) {
+          logger.info("Unable to contact Owner at " + url + ". " + e.getMessage());
+        } else {
+          lastFailure = e;
+          try (StringWriter s = new StringWriter(); PrintWriter w = new PrintWriter(s)) {
+            e.printStackTrace(w);
+            logger.error("Unable to onboard from owner at "
+                    + url + ". " + e.getMessage() + "\n" + s);
+          } catch (IOException e2) {
```

Comment:
Use a proper variable name. ( e2 )

Reply:
Instead of `IOException e2` use `IOException ioException`

Reply:
Using `e` & `e2` variable are also accepted code standards for variable names for exceptions. It has been used at multiple places in the code.